client/components/item/searchItem.js

```diff
@@ -0,0 +1,204 @@
+import {
```

Comment:
This looks good. i think we should make SearchInput.js reusable and utilize it here also